docs/source/scheduling-state.rst

```diff
@@ -84,7 +84,7 @@ forgotten
     dereferenced from the scheduler.
 
 .. note::
-    When the ``distributed.scheduler.worker_saturation`` config value is set to ``inf``
```

Comment:
The documentation itself is still correct, the only thing that changed is the default. If this value is set to `inf`, there is no intermediate state, etc.

However, we changed this and the default is now 1.1 (see section below).

It likely makes sense to rearrange these paragraphs to first talk about a finite value which is the default and then talk about the non-default `inf` value

Reply:
I have moved the paragraph as you suggested.